channels/chan_simpleusb.c

```diff
@@ -3197,6 +3197,36 @@ static void _menu_txb(int fd, struct chan_simpleusb_pvt *o, const char *str)
 	return;
 }
 
+/*!
+ * \brief Update the tune settings to the configuration file.
+ * \param filename	The configuration file being updated (e.g. "simpleusb.conf").
+ * \param category	The category being updated (e.g. "12345").
+ * \param variable	The variable being updated (e.g. "rxboost").
+ * \param value		The value being updated (e.g. "yes").
+ */
+static int tune_variable_update(const char *filename, struct ast_category *category,
+								const char *variable, const char *value)
+{
+	int	res;
+
+	res = ast_variable_update(category, variable, value, NULL, 0);
```

Comment:
Asterisk coding style prefers this kind of control flow:

```
if (failed) {
  return -1;
}
// continue if success
```

There shouldn't need to be more than one level of blocks with this approach, as opposed to up to two now (basically, invert your conditions)

Reply:
Reworked